frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -158,12 +158,14 @@ def simulate(self, *args: Any) -> Any:
 
         exporter = SimulatedValueExporter.new(self.simulator.client_specs.client_parameters)
         exported = [
-            None
-            if arg is None
-            else Value(
-                exporter.export_tensor(position, arg.flatten().tolist(), list(arg.shape))
-                if isinstance(arg, np.ndarray) and arg.shape != ()
-                else exporter.export_scalar(position, int(arg))
+            (
```

Comment:
not sure why, but my `make format` changes some other files

Reply:
black update?

Reply:
https://github.com/zama-ai/concrete/pull/798/files